website/public/css/index.styl

```diff
@@ -110,6 +110,12 @@ html, body
 -------------------------------------------------- */
 //move
 
+.modal-subheading {
+  margin: 12px 0 8px;
+  font-size: 24px;
+  font-weight: bold;
+}
```

Comment:
`.lead` has 20px of bottom margin, which exaggerates the spacing. I'm not sure there's much advantage in changing it, but I'll defer to your preference. Screenshot of your suggested change, for reference:

![image](https://cloud.githubusercontent.com/assets/977577/12633748/fde10cbc-c528-11e5-9cbf-ce115f70789a.png)